src/sign.rs

```diff
@@ -1,23 +1,45 @@
 use std::borrow::Cow;
 use std::io::Write;
 
+use openpgp::armor;
 use openpgp::serialize::stream::Armorer;
 use openpgp::serialize::stream::{LiteralWriter, Message};
 use pyo3::prelude::*;
 use sequoia_openpgp as openpgp;
 
 use crate::signer::PySigner;
 
+#[pyclass(eq, eq_int)]
+#[derive(PartialEq)]
+pub enum SignatureMode {
+    INLINE,
```

Comment:
Could you see if the following would work (it [would seem so](https://github.com/PyO3/pyo3/pull/2457/files#diff-805658921525b2e552e28193b258c8211b58c12ef3c9f5966ef460e5a8afab21R906)):

```suggestion
    #[pyo3(name = "INLINE")]
    Inline,
```

So that the variant on Rust's side would comply with rust naming guidelines but on the Python's side it'd fit Python (so to have a :cookie: and eat it too :wink:).

Reply:
Thanks. I thought about it but didn't know how to do it.